api/api.go

```diff
@@ -768,8 +772,18 @@ func (api *Server) GetEvmTransfersByBlockHeight(ctx context.Context, in *iotexap
 		return nil, status.Error(codes.Unavailable, "evm transfer index not supported")
 	}
 
+	if in.BlockHeight < 1 {
+		return nil, status.Errorf(codes.OutOfRange, "invalid block height = %d", in.BlockHeight)
```

Comment:
To be consistent with other existing APIs, use codes.InvalidArgument here as well.

Reply:
ok